docs/pages/web/popover.js

```diff
@@ -540,128 +545,21 @@ For the \`role\` prop, use:
-                      naturalHeight={50}
-                      naturalWidth={50}
-                      src={data[0]}
-                    />
-                  </Mask>
-                </Box>
-                <Text align="center" color="default" weight="bold">
-                  {data[1]}
-                </Text>
-              </Flex>
-            </TapArea>
-        ))}
-      </Flex>
-    </Flex>
-  );
-
-  return (
-    <React.Fragment>
-      <Flex alignItems="center" gap={{ row: 2, column: 0 }}>
-        <Button
-          accessibilityHaspopup={true}
-          accessibilityExpanded={open}
-          accessibilityControls="example-a11y"
-          color="white"
-          iconEnd="arrow-down"
-          onClick={() => setOpen(!open)}
-          ref={anchorRef}
-          size="lg"
-          selected={open}
-          text={selectedBoard}
-        />
-        <Button color="red" onClick={() => {}} size="lg" text="Save" />
-      </Flex>
-      {open && (
-        <Layer>
-          <Popover
-            accessibilityLabel="Save to board"
-            anchor={anchorRef.current}
-            id="example-a11y"
-            idealDirection="down"
-            onDismiss={() => setOpen(false)}
-            positionRelativeToAnchor={false}
-            role="listbox"
-            size="xl"
-          >
-            <Box width={360}>
-              <Box flex="grow" marginEnd={4} marginStart={4} marginTop={6} marginBottom={8}>
-                <Flex direction="column" gap={{ column: 6, row: 0 }}>
-                  <Text align="center" color="default" weight="bold">
-                    Save to board
-                  </Text>
-                  <SearchBoardField />
-                </Flex>
-              </Box>
-              <Box height={300} overflow="scrollY">
-                <Box marginEnd={4} marginStart={4}>
-                  <Flex direction="column" gap={{ column: 8, row: 0 }}>
-                    <List title="Top choices"/>
-                    <List title="All boards"/>
-                  </Flex>
-                </Box>
-              </Box>
-            </Box>
-          </Popover>
-        </Layer>
-      )}
-    </React.Fragment>
-  );
-}
-  `}
+            sandpackExample={
+              <SandpackExample
+                code={defaultExample}
+                name="Double modal example"
+                hideEditor
+                hideControls
+                previewHeight={PREVIEW_HEIGHT}
+              />
+            }
           />
         </MainSection.Subsection>
       </AccessibilitySection>
       <MainSection
         name="Localization"
-        description="Be sure to localize any text elements within Popover. Note that localization can lengthen text by 20 to 30 percent."
+        description="Be sure to localize any text elements within Popover, along with `accessibilityLabel` and `accessibilityDismissButtonLabel`. Note that localization can lengthen text by 20 to 30 percent."
```

Comment:
Could we add some SlimBanners here and there like in Link? https://gestalt.pinterest.systems/web/link#Localization